tests/unit/amo/components/TestSearchFilters.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+
+import SearchFilters, {
+  SearchFiltersBase,
+} from 'amo/components/SearchFilters';
+import { searchStart } from 'core/actions/search';
+import { ADDON_TYPE_EXTENSION, OS_LINUX } from 'core/constants';
+import { convertFiltersToQueryParams } from 'core/searchUtils';
+import { dispatchClientMetadata } from 'tests/unit/amo/helpers';
+import {
+  createStubErrorHandler,
+  getFakeI18nInst,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  const fakeEvent = { preventDefault: sinon.stub() };
+  let context;
+
+  function render({
+    filters = {},
+    pathname = '/search/',
+    ...props
+  } = {}) {
+    const errorHandler = createStubErrorHandler();
+    const { store } = dispatchClientMetadata();
+
+    store.dispatch(searchStart({ errorHandlerId: errorHandler.id, filters }));
+
+    return shallowUntilTarget(
+      <SearchFilters
+        i18n={getFakeI18nInst()}
+        store={store}
+        pathname={pathname}
+        {...props}
+      />, SearchFiltersBase,
+      { shallowOptions: { context } }
+    );
+  }
+
+  beforeEach(() => {
+    context = { router: { push: sinon.stub() } };
+  });
+
+  it('renders a SearchFilters component', () => {
+    const root = render();
+
+    expect(root.find('.SearchFilters')).toHaveLength(1);
+  });
+
+  it('changes the URL when a new addonType filter is selected', () => {
+    const root = render({ filters: { query: 'Music player' } });
```

Comment:
Thanks for adding a realistic value for `query`. I think that kind of thing helps understand the tests better.